packages/meditrak-app/app/main.js

```diff
@@ -72,3 +72,6 @@ const App = () => (
 );
 
 AppRegistry.registerComponent('TupaiaMediTrak', () => App);
+
+LogBox.ignoreLogs(['Warning: ...']); // Ignore log notification by message
+LogBox.ignoreAllLogs(); // Ignore all log notifications
```

Comment:
Is the annoying yellow box a new thing?

Reply:
This thing at the bottom, just gives PropType warnings, and perfectly covers our survey screen navigation 😆 
![image](https://github.com/beyondessential/tupaia/assets/667652/2e21993c-081e-4dec-845d-9cabdebeeb20)